app/views/offer_mailer/offer_declined.html.erb

```diff
@@ -0,0 +1,8 @@
+<p><%= @sender %> declined your <%= link_to "offer", notification_url(@notification.record) %>:</p>
+
+<blockquote>
+  <p>Start date: <%= @offer.start_date %></p>
+  <p>Pay rate: <%= @offer.pay_rate_value %>/<%= @offer.pay_rate_time_unit_key %></p>
```

Comment:
To prevent mistakes, it would be easier to define a `Offer#rate` method like

```ruby
def rate
  "#{@offer.pay_rate_value}/#{@offer.pay_rate_time_unit_key}"
end
```
as this is used in a few places